src/MovieDisplay.js

```diff
@@ -14,8 +14,12 @@ const MovieDisplay = ( {movie, goHome}) => {
         <img className='moviePoster' src={movie.poster_path} />
         <div className='movieInfo'>
           <p>Title: {movie.title}</p>
-          <p>Average Rating: {movie.average_rating}</p>
+          <p>Average Rating: {movie.average_rating.toFixed(1)}</p>
+          <p>Overview: {movie.overview}</p>
           <p>Release Date: {movie.release_date}</p>
+          <p>Runtime: {movie.runtime}</p>
+          <p>Tagline: {movie.tagline}</p>
+          <p>Genres: {movie.genres.join(', ')}</p>
```

Comment:
Cool use of join to fix how the genres were displaying!

Reply:
Nice work with displaying the details.  If you wanted to, you could do another layer of destructuring here so you don't have to keep doing dot notation for each value.  For example:

```
const { title, average_rating, overview, release_date, runtime, tagline, genres } = movie;

// And then in your JSX:
  <p>Title: { title }</p>
  <p>Average Rating: { average_rating }</p>
```